### PR TITLE
lps-92427-2

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/build.gradle [hunk 1]
```diff
@@ -2,21 +2,36 @@
...
+	compileOnly group: "com.liferay", name: "com.liferay.petra.string"
+	compileOnly group: "com.liferay", name: "com.liferay.portal.aop.api"
 #end
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel"
 #else
-	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.5.0"
+#if (${liferayVersion.startsWith("7.0")} || ${liferayVersion.startsWith("7.1")})
+	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bnd.annotation", version: "@biz.aQute.bnd.version@"
+#end
+	compileOnly group: "com.liferay", name: "com.liferay.osgi.util", version: "3.0.0"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/build.gradle]: This should be 7.2 only.  We didn't need this in 7.0 or 7.1

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/pom.xml [hunk 2]
```diff
@@ -14,53 +14,83 @@
...
-			<artifactId>com.liferay.portal.spring.extender</artifactId>
+			<artifactId>com.liferay.osgi.service.tracker.collections</artifactId>
 			<version>2.0.0</version>
 			<scope>provided</scope>
 		</dependency>
-#elseif (${liferayVersion.startsWith("7.1")})
 		<dependency>
 			<groupId>com.liferay</groupId>
-			<artifactId>com.liferay.portal.spring.extender.api</artifactId>
+			<artifactId>com.liferay.osgi.util</artifactId>
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-service/pom.xml]: This is only 7.2

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
w/ dirty commit history

**Comment by lawrence-lee** [comment]
resent: https://github.com/gamerson/liferay-portal/pull/268

**Pull request closed without merging**